### ISSUE TITLE
Enhancement: Save Model & Predict

### ISSUE THREAD

**Issue by yamcomnet** [ahirner/pytorch-retraining#4]
What would be the best way to save models and then use saved model to make predictions?

**Comment by ahirner** [comment]
You'd get `model.statedict()`and save it with `torch.save(...)`. Ideally, just save the model if it was of greater accuracy than any epoch before. One approach is to wrap the `statedict` in an outer dictionary that also contains this value.
Load the `statedict` with `torch.load(...)` and then into the model with `model.load_state_dict(...)`. Hope that helps. I might bolt that very necessary feature on one day ;)

**Comment by yamcomnet** [comment]
Sounds good approach. I see this package is already very nice for fine tuning.  Would be superb to save models and use those to predict. Nice work!
